types/hooks/screens/useChatUtilsHook.types.ts

```diff
@@ -1,4 +1,6 @@
+import ChannelListSchema from '../../../src/database/schema/ChannelListSchema';
 import {ANONYMOUS, SIGNED} from '../../../src/hooks/core/constant';
+import {AllowedGoToChatScreen} from '../../../src/hooks/core/chat/useChatUtilsHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `ChannelListSchema` has been added, but it seems to be unused in the rest of the code provided. If it's not used elsewhere in the file, it should be removed to avoid unnecessary imports.
```diff
- import ChannelListSchema from '../../../src/database/schema/ChannelListSchema';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->